engine/source/gui/guiFadeinBitmapCtrl.cc

```diff
@@ -93,7 +93,12 @@ class GuiFadeinBitmapCtrl : public GuiBitmapCtrl
       {
          // done state
          alpha = fadeoutTime ? 255 : 0;
-         done = true;
+         if (!done) {

```

Comment:
This coding style makes baby Jesus cry.
And inconsistent with the rest of the file. ;)